### PR TITLE
feat: Add contacts

### LINKED ISSUE #15: 🚀 - add contacts to iot dashboard
### Description

add contacts to iot dashboard

### PR DESCRIPTION
**Pull request by MishaZhem**
Closes #15

### CODE REVIEW

File: apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                    tuiTitle
+                    [style.text-align]="'center'"
+                >
+                    Recent
+                </h2>
+            </header>
+        </button>
+    </tui-tabs>
+    <tui-scrollbar>
+        @if (contacts) {
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html:34]: let's use `if` inside cdk-virtual-scroll

File: apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            <cdk-virtual-scroll-viewport
+                itemSize="50"
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html:36]: seems like itemSize is wrong, items twitching sometimes when scrolling

File: apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.less [hunk 3]
```diff
@@ -0,0 +1,14 @@
+tui-tabs {
+    display: flex;
+    justify-content: space-around;
+}
+
+[tuiCardLarge] [tuiCell][data-size='l'] {
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.less:6]: try to not override card/cell properties, 
remove this, add gap: var(--tui-gap) for list, make bottom-border using `box-shadow`

File: apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+            <header tuiHeader>
+                <h2
+                    tuiTitle
+                    [style.text-align]="'center'"
+                >
+                    Contacts
+                </h2>
+            </header>
+        </button>
+        <button
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html:19]: lets use `for` cycle for tabs
`tabs = ['Contacts', 'Recent']` in component

File: apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html [hunk 5]
```diff
@@ -0,0 +1,87 @@
+<div
+    tuiAppearance="whiteblock"
+    tuiCardLarge="normal"
+>
+    <tui-tabs tuiFade>
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html:5]: let's use `activeItemIndex` https://taiga-ui.dev/next/navigation/tabs#basic
`tuiFade` is redundant

File: apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html (hunk 1 above)
Comment (MishaZhem) [reply apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html:34]: If I do this, the items will only update once

Comment (MishaZhem) [reply apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html:34]: cdkVirtualFor works a little differently

Comment (vladimirpotekhin) [reply apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html]: ok, let's leave it like this for now

File: apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+                >
+                    <tui-avatar
+                        [src]="person.img"
+                        [style.background]="person.img | tuiAutoColor"
+                    />
+                    <div tuiTitle>
+                        {{ person.name }}
+                        <div tuiSubtitle>{{ person.status }}</div>
+                    </div>
+                    <button tuiAppearance="invisible">
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html:41]: `<button tuiIconButton iconStart="@tui.phone" appearance="icon">Call</button>`

File: apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.less [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+.viewport {
+    height: 12.5rem;
+}
+
+tui-badged-content tui-avatar {
+    --t-size: 2.5rem;
+}
+
+[tuiAppearance][data-appearance='invisible'] {
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.less:22]: remove this

File: apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.less [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+    border-radius: 0;
+    box-shadow: 0 1px var(--tui-border-normal);
+}
+
+.viewport {
+    height: 12.5rem;
+}
+
+tui-badged-content tui-avatar {
+    --t-size: 2.5rem;
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.less:19]: we need to make the font for the initials the same too

File: apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html [hunk 9]
```diff
@@ -0,0 +1,80 @@
...
+                        tuiTitle
+                        [style.text-align]="'center'"
+                    >
+                        {{ tab }}
+                    </h2>
+                </header>
+            </button>
+        }
+    </tui-tabs>
+    <tui-scrollbar>
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html:22]: Scrollbar disappears when switching between tabs. Try to put tui-scrollbar under the if{}

File: apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.ts [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+        CdkVirtualScrollViewport,
+        CommonModule,
+        TuiAppearance,
+        TuiAutoColorPipe,
+        TuiAvatar,
+        TuiBadge,
+        TuiBadgedContent,
+        TuiCardLarge,
+        TuiCell,
+        TuiFade,
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.ts:43]: Redundant import

File: apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.ts [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+    templateUrl: './contacts.component.html',
+    styleUrl: './contacts.component.less',
+    changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ContactsComponent {
+    protected contactsService = inject(ContactsService).contactsData;
+    protected tabs = ['Contacts', 'Recent'];
+    protected activeTab = 0;
+
+    protected updateFades(value: number): void {
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.ts:60]: let's remove this method. Just (click)="activeTab = $index" in template

File: apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+                    *cdkVirtualFor="let person of contactsService.recent"
+                    tuiCell="l"
+                >
+                    <tui-badged-content>
+                        <tui-avatar
+                            class="avatar"
+                            [src]="person.img"
+                            [style.background]="person.img | tuiAutoColor"
+                        />
+                        <button
```
Comment (vladimirpotekhin) [inline apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.html:69]: why button? just tui-icon, see https://taiga-ui--pr8168-notification-lj3dbtar.web.app/layout/cell#label

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**